platform/default/src/mbgl/gfx/headless_frontend.cpp

```diff
@@ -139,19 +140,20 @@ PremultipliedImage HeadlessFrontend::readStillImage() {
     return backend->readStillImage();
 }
 
-PremultipliedImage HeadlessFrontend::render(Map& map) {
-    PremultipliedImage result;
+HeadlessFrontend::RenderResult HeadlessFrontend::render(Map& map) {
+    HeadlessFrontend::RenderResult result;
     std::exception_ptr error;
 
     map.renderStill([&](std::exception_ptr e) {
         if (e) {
             error = e;
         } else {
-            result = backend->readStillImage();
+            result.image = backend->readStillImage();
+            result.stats = getBackend()->getContext().renderingStats();
```

Comment:
`backend->getContext()`?

Reply:
`getBackend()` is not a trivial getter function